### PR TITLE
Add simple-matrix-bot-lib to sdks discover page

### PR DESCRIPTION
**Pull request by KrazyKirby99999**

### CODE REVIEW

**Comment by KrazyKirby99999** [comment]
Commit 088de58 has an inaccurate description.

**Comment by MadLittleMods** [comment]
> Commit [088de58](https://github.com/matrix-org/matrix.org/commit/088de5812db01a1860a3b39c9631665a5e8b92fd) has an inaccurate description.

@KrazyKirby99999 I'm confused. You linked the current commit (088de5812db01a1860a3b39c9631665a5e8b92fd) as the one that is inaccurate? Can you update this PR?

**Comment by KrazyKirby99999** [comment]
The changes of the commit are correct, but the commit message should be "
Initial simple-matrix-bot-lib discover page". Sorry for the confusion.

File: gatsby/content/projects/sdks/simple-matrix-bot-lib.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+license: MIT
+home: https://simple-matrix-bot-lib.readthedocs.io/
+room: "#simplematrixbotlib:matrix.org"
+e2e: "no"
+thumbnail: /docs/projects/images/python.png
+screenshot: /docs/projects/images/python.png
+maturity: Released
+---
+
+Simple-Matrix-Bot-Lib is a Python bot library for the Matrix ecosystem built on matrix-nio.
```
Comment (MadLittleMods) [inline gatsby/content/projects/sdks/simple-matrix-bot-lib.mdx:20]: We should add a link here so it's easy for people to reference what is being talked about:

```suggestion
Simple-Matrix-Bot-Lib is a Python bot library for the Matrix ecosystem built on [matrix-nio](https://github.com/poljar/matrix-nio)
```

**Review by MadLittleMods — CHANGES_REQUESTED** [review]
Tested it out locally and looks good. Only a small change suggested 🙂

http://localhost:8000/sdks/#simple-matrix-bot-lib
Screen Shot 2021-07-21 at 11 08 59 PM


http://localhost:8000/docs/projects/sdk/simple-matrix-bot-lib
Screen Shot 2021-07-21 at 11 10 34 PM

**Comment by KrazyKirby99999** [comment]
The suggestion should be accepted now.

**Comment by MadLittleMods** [comment]
Merged! Thanks for adding your project @KrazyKirby99999 🐂

**Pull request merged**